packages/models-library/src/models_library/api_schemas_webserver/licensed_items.py

```diff
@@ -1,25 +1,61 @@
 from datetime import datetime
-from typing import NamedTuple, Self
+from typing import Any, NamedTuple
 
-from common_library.dict_tools import remap_keys
 from models_library.licensed_items import (
-    LicensedItemDB,
+    VIP_DETAILS_EXAMPLE,
     LicensedItemID,
     LicensedResourceType,
 )
 from models_library.resource_tracker import PricingPlanId
-from pydantic import ConfigDict, PositiveInt
+from models_library.utils.common_validators import to_camel_recursive
+from pydantic import AfterValidator, BaseModel, ConfigDict, PositiveInt
+from typing_extensions import Annotated
 
 from ._base import OutputSchema
 
+# RPC
 
-class LicensedItemGet(OutputSchema):
-    licensed_item_id: LicensedItemID
 
-    name: str
-    license_key: str | None
+class LicensedItemRpcGet(BaseModel):
+    licensed_item_id: LicensedItemID
+    licensed_resource_name: str
     licensed_resource_type: LicensedResourceType
+    licensed_resource_data: dict[str, Any]
```

Comment:
I would try the discriminators approach for the licensed_resource section here

https://docs.pydantic.dev/latest/concepts/unions/#exactness